File: PruebaPacientes/PruebaPacientes/EnfermedadesPreviasComorbilidades.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Modelo
{
    public class EnfermedadesPreviasComorbilidades
    {
        public int Id                                   { get; set; }

        public bool PresentaEnfermedadesComorbilidades  { get; set; }

        public bool InmunoSupresionCongenitaOAdquirida  { get; set; }
        public bool Diabetes                            { get; set; }
        public bool Obesidad                            { get; set; }
        public bool Embarzo                             { get; set; }
        public bool Puerperio                           { get; set; }
        public bool Prematuridad                        { get; set; }
        public string SemanasDePrematuridad             { get; set; }
        public bool BajoPesoAlNacer                     { get; set; }
        public string CantidadDePesoAlNacer             { get; set; }
        public bool EnfermedadNeurologica               { get; set; }
        public bool EnfermedadHepatica                  { get; set; }
        public bool EnfermedadRenalCronica              { get; set; }
        public bool HipertensionArterial                { get; set; }
        public bool EnfermedadOncologica                { get; set; }
        public bool BronquitisPrevia                    { get; set; }
        public bool NAC_Previa                          { get; set; }
        public bool EPOC                                { get; set; }
        public bool Asma                                { get; set; }
        public bool Tubercolosis                        { get; set; }
        public bool Fumador                             { get; set; }
        public bool NingunaDelasAnteriores              { get; set; }
        public bool Otras                               { get; set; }
        public string OtrasDetalle                      { get; set; }
        //public int FormularioParaAnalisisId { get; set; }
        //public FormularioParaAnalisis FormularioParaAnalisis { get; set; }


    }
}

File: Frontend/api/models/EnfermedadesPreviasComorbilidades.js
/**
 * EnfermedadesPreviasComorbilidades.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {
    PresentaEnfermedadesComorbilidades:{
      type: 'boolean',
      defaultsTo: false
    },
    InmunoSupresionCongenitaOAdquirida:{
      type: 'boolean',
      defaultsTo: false
    },
    Diabetes:{
      type: 'boolean',
      defaultsTo: false
    },
    Obesidad:{
      type: 'boolean',
      defaultsTo: false
    },
    Embarazo:{
      type: 'boolean',
      defaultsTo: false
    },
    Puerperio:{
      type: 'boolean',
      defaultsTo: false
    },
    Prematuridad:{
      type: 'boolean',
      defaultsTo: false
    },
    SemanasDePrematuridad:{
      type: 'string'
    },
    BajoPesoAlNacer:{
      type: 'boolean',
      defaultsTo: false
    },
    CantidadPesoAlNacer:{
      type: 'string'
    },
    EnfermedadNeurologica:{
      type: 'boolean',
      defaultsTo: false
    },
    EnfermedadHepatica:{
      type: 'boolean',
      defaultsTo: false
    },
    EnfermedadRenalCronica:{
      type: 'boolean',
      defaultsTo: false
    },
    HipertensionArterial:{
      type: 'boolean',
      defaultsTo: false
    },
    EnfermedadOncologica:{
      type: 'boolean',
      defaultsTo: false
    },
    BronquitisPrevia:{
      type: 'boolean',
      defaultsTo: false
    },
    NAC_Previa:{
      type: 'boolean',
      defaultsTo: false
    },
    EPOC:{
      type: 'boolean',
      defaultsTo: false
    },
    Asma:{
      type: 'boolean',
      defaultsTo: false
    },
    Tuberculosis:{
      type: 'boolean',
      defaultsTo: false
    },
    Fumador:{
      type: 'boolean',
      defaultsTo: false
    },
    NingunaDeLasAnteriores:{
      type: 'boolean',
      defaultsTo: false
    },
    Otras:{
      type: 'boolean',
      defaultsTo: false
    },
    OtrasDetalle:{
      type: 'string'
    },
    //Relacion
    owner:{
      model:'FormularioParaAnalisis',
      unique:true
    }
  },

};

